downstream/titles/hub/managing-user-access/docinfo.xml

```diff
@@ -1,10 +1,10 @@
 <title>Managing user access in Private Automation Hub</title>
 <productname>Red Hat Ansible Automation Platform</productname>
-<productnumber>1.2</productnumber>
-<subtitle>Configure Automation Hub to support your organization by creating groups for your users and providing them with the level of system access they require.</subtitle>
+<productnumber>2.1</productnumber>
+<subtitle>Configure automation hub to support your organization by creating groups for your users to provide them with appropriate system permissions, or by allowing view-only access to unauthorized users.</subtitle>
```

Comment:
This sentence is a bit difficult to read in the large uppercase font under the chapter title. Would it make sense to break it up or rephrase?   If it will be rendered in a more readable font in the final docs, disregard this comment.

Reply:
![image](https://user-images.githubusercontent.com/44700011/155768362-0f74f275-af94-41a4-90c5-cd9696cd07ca.png)

Reply:
It's a good point. In the final build it looks like it turns into lowercase, but I would agree that it's a long sentence to begin with. Will cut this sentence down